src/models/db/account.js

```diff
@@ -83,7 +83,9 @@ class Account extends Model {
     // some databases store booleans as 0 and 1, and knex does not convert
     data.is_disabled = Boolean(data.is_disabled)
     data.is_admin = Boolean(data.is_admin)
-    data.balance = Number(data.balance)
+    // oracle has balance stored properly, but knex returns it with a small
+    // rounding error, possibly a bug in knex; using toFixed as workaround
+    data.balance = Number(data.balance.toFixed(2))
```

Comment:
I think the database was already enforcing this constraint. Though it looks like the integration tests are failing due to this.